addons/sales_team/views/crm_team_views.xml

```diff
@@ -117,16 +108,12 @@
         <field name="arch" type="xml">
             <kanban class="o_kanban_mobile" sample="1">
                 <templates>
-                    <t t-name="kanban-box">
-                        <div t-attf-class="oe_kanban_content oe_kanban_global_click">
-                            <div class="row">
-                                <div class="col-6">
-                                    <strong><field name="name"/></strong>
-                                </div>
-                                <div class="col-6">
-                                    <span class="float-end"><field name="user_id"/></span>
-                                </div>
-                            </div>
+                    <t t-name="kanban-card" class="row g-0">
+                        <div class="col-6">
+                            <field name="name" class="fw-bold"/>
+                        </div>
+                        <div class="col-6">
```

Comment:
same here

Reply:
here we need div, otherwise float-end don't work